UI/window-basic-main.cpp

```diff
@@ -2670,6 +2670,20 @@ void OBSBasic::CreateHotkeys()
 	LoadHotkeyPair(replayBufHotkeys, "OBSBasic.StartReplayBuffer",
 		       "OBSBasic.StopReplayBuffer");
 
+	auto replayBufferCallback = [](void *data, obs_hotkey_id,
+				       obs_hotkey_t *, bool pressed) {
+		OBSBasic &basic = *static_cast<OBSBasic *>(data);
+		if (basic.outputHandler->ReplayBufferActive() && pressed) {
+			blog(LOG_INFO, "Saving replay buffer due to hotkey");
+			basic.ReplayBufferSave();
+		}
+	};
+
+	replayBufSaveHotkey = obs_hotkey_register_frontend(
+		"OBSBasic.SaveReplayBuffer", Str("Basic.Main.SaveReplay"),
+		replayBufferCallback, this);
```

Comment:
Are you *sure* that existing hotkeys are preserved here?
Not only is the registering function now _frontend instead of _output, but the id is also changed from ReplayBuffer.Save to OBSBasic.SaveReplayBuffer.

Reply:
resolved by checking on hotkey load whether "new" key data exists, and if not, attempting to load "old" key data.